src/chi-vue/src/constants/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const ANIMATION_DURATION = {
```

Comment:
This is a perfect way of maintaining grouped consts. And you did it right with the capital letters. 🥳